avocado/core/messages.py

```diff
@@ -14,6 +14,23 @@
 
 import os
 
+from .test_id import TestID
+
+
+def _convert_identifier_to_test_id(identifier):
+    """
+    It converts Task identifier to TestID
+
+    :param identifier: Task identifier
+    :type: :class:`avocado.core.nrunner.Task.identifier`
```

Comment:
nitpicking: Today, task.identifier has not "type" defined. Can be TestID or string. So one possibility, is that this type could be pointing directly to the end types.

Reply:
Yes I wasn't sure with this. I wanted to point out that it is related to the Task.